src/scss/_custom.scss

```diff
@@ -1,3 +1,16 @@
-p {
-  color: red;
+.leftcol {
+  background-color: blue;
+  float: left;
+  clear: right;
+  display: inline-block;
+  width: 30%;
+  margin: 10px;
+}
+
+.rightcol {
+  background-color: green;
+  float: left;
+  display: inline-block;
```

Comment:
Honestly, the `display: inline-block` property and `float: left` property are somewhat redundant. _Either_ float the columns or use `inline-block`, although the better choice here is `float`, I think.